nautobot/core/templates/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'base.html' %}
+{% load helpers %}
+{% load registry %}
+
+{% block header %}
+    {{ block.super }}
+    {% if new_release %}
+        {# new_release is set only if the current user is a superuser or staff member #}
+        <div class="alert alert-info text-center" role="alert">
+            <i class="mdi mdi-information-outline"></i>
+            A new release is available: <a href="{{ new_release.url }}">Nautobot v{{ new_release.version }}</a> |
+            <a href="https://docs.nautobot.com/projects/core/en/v{{ new_release.version }}/installation/upgrading/">Upgrade instructions</a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+
+{% block content %}
+    <h1>About Nautobot</h1>
+    <div class="row">
+        <div class="col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>About</strong>
+                </div>
+                <div class="panel-body">
+                    <a href="https://networktocode.com/nautobot/">Nautobot</a> is a product of <a href="https://networktocode.com/">Network to Code</a>. Nautobot is a Network Source of Truth and Network Automation Platform. Nautobot has an extensive ecosystem of solutions that can be installed to provide power Network Automation feature sets. Please checkout the <a href="{% url 'apps:apps_marketplace' %}">App Marketplace</a> to learn more.
+                </div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Support</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Support Contract Status</td>
+                        <td>{{ support_contract_active | render_boolean }}</td>
+                    </tr>
+                    <tr>
+                        <td>Contract Expiry Date</td>
```

Comment:
`Support Contract Status` versus `Contract Expiry Date` feels a bit inconsistent to me - should both or neither label contain the `Support` keyword?

Reply:
@jedelman8

Reply:
We think no for now.